src/mobility/components/BicycleSheet.tsx

```diff
@@ -71,7 +75,7 @@ export const BicycleSheet = ({vehicleId: id, close}: Props) => {
         )}
         {!isLoading && !isError && vehicle && (
           <>
-            <ScrollView style={style.container}>
+            <View style={style.container}>
```

Comment:
Needs to be `ScrollView`. Test it with larger text sizes to see why.

Reply:
reverted to ScrollView